coreos-base/oem-interoute/files/cloud-config.yml

```diff
@@ -0,0 +1,49 @@
+#cloud-config
+
+coreos:
+    units:
+      - name: vmtoolsd.service
+        command: start
+        content: |
+          [Unit]
+          Description=VMware Tools Agent
+          Documentation=http://open-vm-tools.sourceforge.net/
+          ConditionVirtualization=vmware
+
+          [Service]
+          ExecStartPre=/usr/bin/ln -sfT /usr/share/oem/vmware-tools /etc/vmware-tools
```

Comment:
Our hypervisor is VMware. We need the scaling stuff.

-----Original Message-----
From: "Alex Crawford" <notifications@github.com>
Sent: ‎04/‎09/‎2015 19:48
To: "coreos/coreos-overlay" <coreos-overlay@noreply.github.com>
Cc: "Dong Xie" <xied75@gmail.com>
Subject: Re: [coreos-overlay] init commit of interoute oem files (#1511)

In coreos-base/oem-interoute/files/cloud-config.yml:
> @@ -0,0 +1,49 @@
> +#cloud-config
> +
> +coreos:
> +    units:
> +      - name: vmtoolsd.service
> +        command: start
> +        content: |
> +          [Unit]
> +          Description=VMware Tools Agent
> +          Documentation=http://open-vm-tools.sourceforge.net/
> +          ConditionVirtualization=vmware
> +
> +          [Service]
> +          ExecStartPre=/usr/bin/ln -sfT /usr/share/oem/vmware-tools /etc/vmware-tools
Why is this here?
—
Reply to this email directly or view it on GitHub.